docs/LogGuide.md

```diff
@@ -2453,3 +2461,114 @@ Parsed Log Line Examples:
 ```
 
 <!-- AUTO-GENERATED-CONTENT:END (logLines:type=InCombat&lang=en-US) -->
+
+<a name="line261"></a>
+
+### Line 261 (0x105): CombatantMemory
+
+OverlayPlugin has a `getCombatants` function which returns the state of combatants in the game.
+However, it is hard to know when it is safe to call this function
+and know that combatants have moved into position (or changed model or changed heading).
+These lines give more granular information when combatants change their status.
+Please note that this is still polling memory (so timing may be racy)
+and there are some heuristics to not emit too many lines (so data may be imprecise).
```

Comment:
It may make sense to link back to OP's code for the heuristics here? Maybe something like:

```md
For more information, see the [class definition](https://github.com/OverlayPlugin/OverlayPlugin/blob/main/OverlayPlugin.Core/MemoryProcessors/Combatant/LineCombatant.cs#L27) in OverlayPlugin
```

Reply:
Done, thanks!